uint8 index
int32 autobias
---
int32 error